sources/PseudoTerminal.m

```diff
@@ -5686,11 +5688,14 @@ - (void)_refreshTerminal:(NSNotification *)aNotification {
 
         // Update dimmed status of inactive sessions in split panes in case the preference changed.
         for (PTYSession* aSession in [aTab sessions]) {
-                        [self setDimmingForSession:aSession];
+                        [self updateDimAndBlurForSession:aSession];
             [[aSession view] setBackgroundDimmed:![[self window] isKeyWindow]];
 
             // In case dimming amount slider moved update the dimming amount.
             [[aSession view] updateDim];
+            
+            
```

Comment:
Please remove one of the empty lines (two consecutive newlines aren't used except between methods)